src/main/java/core/basesyntax/strategy/impl/ReturnOperationHandler.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.enums.Operation;
+import core.basesyntax.strategy.QuantityIncrementer;
+import java.util.Map;
+
+public class ReturnOperationHandler
+        extends AbstractOperationHandler
+        implements QuantityIncrementer {
+    public ReturnOperationHandler() {
```

Comment:
According to the checklist, don't begin class or method implementation with an empty line. Please remove the empty line at the beginning of the class constructor.